package_manager/dpkg_extract.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+set -o errexit
+
+DEB=$1
+shift
```

Comment:
I get `line 6: shift: shift count out of range` when I don't provide any argument to the script. (You may see different behaviors when using a `sh` vs `bash`.)

Reply:
I also have a python version of this script in another personal project that includes a basic form of filtering (excluding) files. That script is also using the build_tar instead of the unix tooling. 

It was developed after I opened this PR, so it might need some polish (an hour or so).

Might be better to switch to that one instead of fixing this shell script? Or do you have a preference for shell scripts?

Reply:
I think a shell script is fine. Not that a script is a good thing, but I don't think we want over-engineering. I mean, the goal is to not increase complexity and code lines to reduce maintenance burden. When I first came to this repo, everything was overwhelming.

Reply:
But the excluding feature will probably be a nice thing that we may need anyway. But let's not do it here to keep the PR small. We can always do it later.

Reply:
Should be fixed